code/controllers/subsystem/dcs.dm

```diff
@@ -0,0 +1,32 @@
+SUBSYSTEM_DEF(dcs)
```

Comment:
SSdcs doesn't roll off the brain as easily as SScomponents but that's just my opinion

Reply:
I felt calling it components would be misleading since this doesn't really have anything to do with managing the actual components